centinel/primitives/tls.py

```diff
@@ -22,6 +23,11 @@ def get_fingerprint(host, port=443, external=None):
         tls_error = str(exp)
         exception = exp
 
+    # this comes out as unicode, but m2crypto breaks if it gets
+    # something other than a string, so convert to ascii
+    if type(cert) == unicode or test:
```

Comment:
woops, I was moving on too quickly from testing. I need to remove that and repush.